### ISSUE TITLE
Return the image in undo/redo calls

### ISSUE THREAD

**Issue by theawless** [leader22/simple-drawing-board.js#21]
One use case would be to send the image just after an undo/redo call.
Currently, because of the async nature of _setImgByImgSrc we cannot do this:
```
    pad.undo();
    server.send(pad.getImg()); // img might still not be updated
```

A proposed fix would be to return the image in the call:
```
    const img = pad.undo();
    server.send(img);
```

I will be able to contribute for this change, do you think this feature is a good idea?

**Comment by leader22** [comment]
I confirmed this issue.

And I want to fix this by making `undo()` and `redo()`(and `_setImgByImgSrc`) as `async` in next major release version.
Does this change satisfy your needs?

**Comment by theawless** [comment]
Yes that should work perfectly.

**Comment by leader22** [comment]
Sure!

Now I start working on and hope to release it in this week.

**Comment by leader22** [comment]
Hi @theawless 

I've finished almost all todos for `v3`.
Since it is a major version up, there are some breaking changes.

Would you please check v3 [README](https://github.com/leader22/simple-drawing-board.js/tree/v3)?
If it looks good to you, I will merge and release it.

**Comment by theawless** [comment]
Hey the readme looks perfect to me. Thanks for adding all these features!

**Comment by leader22** [comment]
Thank you for your review 😉 

Now I've published `v3.0.0`!(w/ some naming fix)
